README.md

```diff
@@ -85,5 +85,86 @@ iex> EchoClient.echo(pid, "Hi")
 *DBG* #PID<0.379.0> received frame: {:text, "Hi"}
 ```
 
+#### start_link/3 && start/3 
+
+You can start a supervisioned WebSockex connection by using the regular **start_link/3**:
+
+```elixir
+iex> {:ok, pid} = WebSockex.start_link(url, __MODULE__, state)
+```
+
+And an unsupervisioned one (meaning it won't try to re-establish the connection on an exceptional close) with **start/3**:
+
+```elixir
+iex> {:ok, pid} = WebSockex.start(url, __MODULE__, state)
+```
+
+This can be useful if you have don't want the socket to automatically try reconnection, you have no need for the built-in supervision or if you're implementing your own supervision strategy.
+
+
```

Comment:
Excess white space I would probably eventually delete. 😝

Reply:
Ah - I didn't realize on emacs I had added empty lines - I will clean up that